integreat_cms/cms/templates/pages/page_pdf.html

```diff
@@ -75,24 +74,25 @@ <h1 id="title_page">{{ language.table_of_contents }}</h1>
             <div><pdf:nextpage></div>
         {% endif %}
 
-        {% recursetree pages %}
-            <li>
-                {% get_public_translation node language.slug as page_translation %}
-                {% if node.level < 2 %}
-                    <h1 class="{% if node.level == 0 %}root_title{% endif %}">{{ page_translation.title }}</h1>
-                {% else %}
-                    <h2>{{ page_translation.title }}</h2>
-                {% endif %}
-                <div class="content" style="padding-bottom: 30px;">
-                    {{ page_translation.text|safe }}
-                </div>
-                {% if not node.is_leaf_node %}
-                    <ul class="children">
-                        {{ children }}
-                    </ul>
-                {% endif %}
-            </li>
-        {% endrecursetree %}
+        {% for page, info in pages.get_annotated_list %}
+            {% if info.open %}
+                <ul><li>
+            {% else %}
+                </li><li>
+            {% endif %}
+            {% get_public_translation page language.slug as page_translation %}
+            {% if page.depth < 3 %}
+                <h1 class="{% if page.depth == 1 %}root_title{% endif %}">{{ page_translation.title }}</h1>
+            {% else %}
+                <h2>{{ page_translation.title }}</h2>
+            {% endif %}
+            <div class="content" style="padding-bottom: 30px;">
+                {{ page_translation.text|safe }}
+            </div>
+            {% for close in info.close %}
+                </li></ul>
+            {% endfor %}
+        {% endfor %}
```

Comment:
Good catch, thanks! I used the method wrong, because it is defined only on the `Page` model and not on the `PageQueryset` model. But now it should be fixed.